test/integration/supervisor_login_test.go

```diff
@@ -221,7 +244,7 @@ func TestSupervisorLogin_Browser(t *testing.T) {
 			spec.Host, env.SupervisorUpstreamActiveDirectory.BindUsername,
 			secret.Name, secret.ResourceVersion,
 		)
-		requireSuccessfulActiveDirectoryIdentityProviderConditions(t, adIDP, expectedMsg)
+		requireSuccessfulActiveDirectoryIdentityProviderConditions(t, adIDP, expectedMsg, env.SupervisorUpstreamActiveDirectory.CABundle != "")
```

Comment:
Is there a better way to do this? I made this change to get us to green CI, but I don't like how this line of code (and other similar lines) is making assumptions about how the `CreateTestActiveDirectoryIdentityProvider` helper (and similar helpers) is working internally.

Reply:
We looked at this and decided to avoid refactoring it for now.